src/main/common/filter.h

```diff
@@ -54,20 +63,29 @@ typedef struct firFilterDenoise_s {
 } firFilterDenoise_t;
 
 typedef struct fastKalman_s {
-    float q;       // process noise covariance
-    float r;       // measurement noise covariance
-    float p;       // estimation error covariance matrix
-    float k;       // kalman gain
+    float k;       // "kalman" gain
     float x;       // state
     float lastX;   // previous state
 } fastKalman_t;
 
+typedef struct laggedMovingAverage_s {
+    uint8_t movingWindowIndex;
+    uint8_t windowSize;
+    float weight;
+    float movingSum;
+    float buf[MAX_LMA_WINDOW_SIZE];
+} laggedMovingAverage_t;
+
 typedef enum {
     FILTER_PT1 = 0,
     FILTER_BIQUAD,
+#if defined(USE_GYRO_FIR_FILTER_DENOISE)
```

Comment:
Not sure if the conditional in here is a good idea, since this enum is used in a parameter group. At least when we start communicating to the configurator with parameter groups (or add `gyro_lowpass_type` to an MSP command), this will force the configurator to know if a particular target has `USE_GYRO_FIR_FILTER_DENOISE` defined.

Better to just put an empty string into this position in the CLI lookup if `USE_FIR_FILTER_DENOISE` isn't defined.

Reply:
This is a wild one, removing (4027bef)